README.md

```diff
@@ -2,18 +2,39 @@
 
 This repo contains all of the challenges from [**r/dailyprogrammer**](http://reddit.com/r/dailyprogrammer) and also scripts used to pull challenges from the subreddit 3 times per week and to cleanup directories & remove unnecessary posts.
 
+## Requirements
+
+- Python 3
+  - praw
+  - pprint
+
+## Installation
+
+  ```bash
+  $ pip3 install -r etc/requirements.txt
+  ```
+
 ## How to use
-The script can also be run via the command line by running `./post_challenges <number_of_challenges>`
 
-### Required Modules:
-- praw
-- subprocess
-- re
-- os
-- pprint
+The script can also be run via the command line by running `./post-challenges.py <number_of_challenges>`
+
+You may wish to run this in a temporary directory, to avoid adding folders directly to the top directory of the repo. Move the various levels of challenges into their respective end points.
+
+### Example full run
 
-**post-challenges.py runs on Python 3.x**
+  ```bash
+  $ cd DailyProgrammerChallenges
+  $ mkdir tmp
```

Comment:
What are we using `tmp/` for here? From what you added above, it looks like we should `cd` into it.

Reply:
Right you are.  Also added a bit about using praw.ini